apps/server/lib/google-drive/src/internal/utils/log.ts

```
@@ -0,0 +1,124 @@
+		return logger[fnLevel].bind(logger);
+	}
+}
+
+const noopLogger = {
+	error: noop,
+	warn: noop,
+	info: noop,
+	debug: noop,
+};
+
+let cachedLoggers = new WeakMap<Logger, [LogLevel, Logger]>();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'cachedLoggers' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)